pyproject.toml

```diff
@@ -42,10 +42,20 @@ geoh5py = {version = "~0.8.0rc3", source = "pypi", allow-prereleases = true}
 geoapps-utils = { version = "~0.2.0a2", source = "pypi", allow-prereleases = true}
 
 ## indirect dependencies, forcing them here for installation through Conda not pip
-h5py = "*" # from geoh5py
-Pillow = "*" # from geoh5py
+h5py = "^3.2.1"  # from geoh5py
+Pillow = "^10.0.1"  # from geoh5py
```

Comment:
if not forcing the version, they sometimes diverge, ending up with one installed by conda and another one with pip through geoph5py